### PR TITLE
Contracts - upgrade solidity compiler version to ^0.4.19

### PR DESCRIPTION
**Pull request by loredanacirstea**
closes https://github.com/raiden-network/microraiden/issues/276

### CODE REVIEW

File: contracts/contracts/RaidenMicroTransferChannels.sol [hunk 1]
```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.17;
+pragma solidity ^0.4.19;
```
Comment (LefterisJP) [inline contracts/contracts/RaidenMicroTransferChannels.sol]: are you using some 0.4.18/0.4.19 only features? If not then I guess this can stay as is.

Comment (loredanacirstea) [reply contracts/contracts/RaidenMicroTransferChannels.sol]: No, `0.4.17` should work too. I usually update this too keep up with https://etherscan.io/solcbuginfo . There is a fixed bug in 0.4.18 but it is of very low severity, though.

Comment (LefterisJP) [reply contracts/contracts/RaidenMicroTransferChannels.sol]: These info appear on the verified contract, depending on the version you compiled the contract with and not the minimum required version for compiling shown in the source code.

**Comment by loredanacirstea** [comment]
Closing this as it is not needed.

**Pull request closed without merging**